src/styles/sidebar/components/help-panel.scss

```diff
@@ -1,45 +1,71 @@
 .help-panel {
-  @include font-normal;
-  background: $grey-3;
-  margin-bottom: 0.72em;
-  padding: $layout-h-margin;
-  border-radius: 2px;
-}
+  &__sub-panel-title {
+    margin: 0;
+    padding: 0.5em;
+    text-align: center;
+    font-size: 1.25em;
+    font-weight: 600;
+  }
 
-.help-panel-title {
-  color: $grey-6;
-  font-weight: bold;
-  // Margin between top of the dialog and
-  // top of x-height of title should be ~15px.
-  margin-top: -5px;
-}
+  &__content {
+    padding: 0.5em;
+    border-top: 1px solid $grey-3;
+    border-bottom: 1px solid $grey-3;
+    line-height: $normal-line-height;
+    font-size: $normal-font-size;
+  }
 
-.help-panel-content {
-  // Margin between bottom of ascent of title and
-  // top of x-height of content should be 20px.
-  margin-top: 11px;
-}
+  &__icon {
+    width: 12px;
+    height: 12px;
+  }
 
-.help-panel-content__key {
-  width: 100px;
-  float: left;
-  color: $grey-4;
-}
+  &__footer {
+    padding: 0.5em 0;
+    display: flex;
+    align-items: center;
+  }
 
-.help-panel-content__val {
-  word-wrap: break-word;
-  margin-left: 100px;
-}
+  &__sub-panel-link {
+    display: flex;
+    align-items: center;
+    color: $brand;
 
-.help-panel-content {
-  margin-top: 10px;
-  margin-bottom: 15px;
-}
+    &--right {
+      margin-left: auto;
+    }
+
+    &-icon {
+      margin: 5px;
+      width: 12px;
+      height: 12px;
+    }
+  }
+
+  &-tabs {
+    display: flex;
+    align-items: center;
+
+    &__tab {
+      flex: 1 1 0px;
+      margin-top: 0.5em;
+      border-right: 1px solid $grey-3;
+      text-align: center;
+      font-size: 1.25em;
+      color: $grey-5;
+
+      &:last-of-type {
+        border-right: none;
+      }
+    }
+
+    &__link {
+      color: $grey-5;
+    }
 
-.help-panel-content__link {
-  color: $grey-6;
-  text-decoration: underline;
-  &:hover {
-    text-decoration: underline;
+    &__icon {
```

Comment:
I see this rule above also. Do we need both or can both elements use `help-panel__icon` ?

Reply:
I like the fake namespacing because `HelpPanelTab` is technically a sub-component (i.e. different component) than `HelpPanel`.